docs/quick_start.rst

```diff
@@ -230,27 +236,124 @@ Connect the new application to the OMF Connector Relay by selecting the new Fled
 
 Finally, select the new Fledge application. Click "More" at the bottom of the Configuration panel. Make note of the Producer Token and Relay Ingress URL.
 
-Now go to the Fledge user interface, create a new North instance and select the “pi_server” plugin on the first screen.
+Now go to the Fledge user interface, create a new North instance and select the “OMF” plugin on the first screen.
+The second screen will request the following information:
+
++---------------------+
+| |omf_plugin_config| |
++---------------------+
+
+- Basic Information
+   - **Endpoint:** Select what you wish to connect to, in this case the Connector Relay.
+   - **Server hostname:** The hostname or address of the Connector Relay.
+   - **Server port:** The port the Connector Relay is listening on. Leave as 0 if you are using the default port.
+   - **Producer Token:** The Producer Token provided by PI
+   - **Data Source:** Defines which data is sent to the PI Server. The readings or Fledge's internal statistics.
+   - **Static Data:** Data to include in every reading sent to PI.  For example, you can use this to specify the location of the devices being monitored by the Fledge server.
+- Connection management (These should only be changed with guidance from support)
+   - **Sleep Time Retry:** Number of seconds to wait before retrying the HTTP connection (Fledge doubles this time after each failed attempt).
+   - **Maximum Retry:** Maximum number of times to retry connecting to the PI server.
+   - **HTTP Timeout:** Number of seconds to wait before Fledge will time out an HTTP connection attempt.
+- Other (Rarely changed)
+   - **Integer Format:** Used to match Fledge data types to the data type configured in PI. This defaults to int64 but may be set to any OMF data type compatible with integer data, e.g. int32.
+   - **Number Format:** Used to match Fledge data types to the data type configured in PI. The defaults is float64 but may be set to any OMF datatype that supports floating point values.
+   - **Compression:** Compress the readings data before sending it to the PI System.
+
+PI Web API OMF Endpoint
+-----------------------
+
+To use the PI Web API OMF endpint first  ensure the OMF option was included in your PI Server when it was installed.  
+
+Now go to the Fledge user interface, create a new North instance and select the “OMF” plugin on the first screen.
+The second screen will request the following information:
+
++---------------------+
+| |omf_plugin_config| |
```

Comment:
This image has with endpoint `OSI Cloud Services` selected. It should be with `PI Web API`